### ISSUE TITLE
Start Menu

### ISSUE THREAD

**Issue by ethanlabelle** [PWhiddy/PokemonRedExperiments#158]
In the interactive mode, how do I open the start menu? as in the menu opened by pressing `start` on GBA. I have noticed the AI looping in the start menu. This may be solved with some hypervisor like logic preventing the player from entering the pokedex or options, etc.

**Comment by niazally** [comment]
You can enter the start menu by pressing "Enter". I encountered this same scenario where the AI randomly enters the start menu and just scrolls through the options.

You can assist the AI - if preferred  - by pressing s (equivalent to B on GBA) to exit the menu.

**Comment by ethanlabelle** [comment]
Yeah I have been doing this while watching some runs. During training, the AI must waste hours in start menus. This issue is similar to #159
